CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
+## Next release
+
+- Add API call for show track chooser
```

Comment:
Could you add a pointer what this API is called? Maybe something like `hgApi. showTrackChooser()`?

Reply:
Yup 👍